ninja-core/src/main/java/ninja/Context.java

```diff
@@ -107,6 +107,13 @@ private HTTP_STATUS(int code) {
     @Deprecated
     String getRequestUri();
 
+    /**
+     * Returns the parameter string that is contained in the request URL after the path.
+     *
+     * @return The parameter string
+     */
+    String getRequestParameterString();
```

Comment:
Could you give an example in the Javadoc? Then it is even more straight-forward for everyone what this method does :)

Reply:
Done